libopae/src/reconf.c

```diff
@@ -295,6 +366,13 @@ fpga_result __FPGA_API__ fpgaReconfigureSlot(fpga_handle fpga,
 		FPGA_ERR("Failed to clear port errors.");
 	}
 
+	if (!(flags & FPGA_RECONF_FORCE)) {
```

Comment:
Here you are using the `flags` param but on line 344 its included in the `UNUSED_PARAM` macro. I think we should delete this correct?

Reply:
Yes. I will make the appropriate changes.